### PR TITLE
Added a new line for the email input field

### LINKED ISSUE #354: Email input field should begin on a new line
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/meilix

**Actual Behaviour**
The present email input field begins on the same line which says that "Fields with * after it are required". Please make it on a new line.

**Expected Behaviour**
The email input field should begin from a new line

**Would you like to work on the issue?**
No, it is for GCI students only

### PR DESCRIPTION
**Pull request by sahilsaha7773**
###  Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes (provide a screenshot or link for test)
- [x]  I have added tests that prove my fix is effective or that my feature works
- [x]  I have added necessary documentation (if appropriate)
### Short description of what this resolves:
 
Added a new line for the email input field.

### Changes proposed in this pull request:

- [Preview link](https://tranquil-thicket-43691.herokuapp.com/)

Fixes #354 

### Screenshots

**Before:**

e

**After:**

ff

--------------------------------------------------------------------------

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
       <div class="row mt-3">
         <div class="col-md-4 mx-auto">
           <div class="form-group mouse-over-text">
-            <label class="heading" for="bookmark">Fields with * marked after them are required<span>
+            <label class="heading" for="bookmark">Fields with * marked after them are required<br><span>
```
Comment (pythongiant) [inline templates/index.html]: add the `<br>`  after the ending `<span>` tag

Comment (sahilsaha7773) [reply templates/index.html]: done

**Comment by sahilsaha7773** [comment]
@sarnava1 Done!!

File: templates/index.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
       <div class="row mt-3">
         <div class="col-md-4 mx-auto">
           <div class="form-group mouse-over-text">
-            <label class="heading" for="bookmark">Fields with * marked after them are required<span>
+            <label class="heading" for="bookmark">Fields with * marked after them are required<span><br><br>
```
[the remark this answers is not in the document]
Comment (sahilsaha7773) [reply templates/index.html]: Done!!

**Comment by sahilsaha7773** [comment]
Please review now @pranav1698

**Comment by sahilsaha7773** [comment]
@pranav1698 ok I am doing it

**Comment by pranav1698** [comment]
Rests looks good to me

**Comment by sahilsaha7773** [comment]
Done!!

**Comment by sarnava1** [comment]
@tabesin @meets2tarun please merge this PR.

**Comment by sahilsaha7773** [comment]
@abishekvashok I did the same first but was requested by @pranav1698 to do it in some other way

**Comment by pranav1698** [comment]
I thought that external css added is better than br tags in the html code
Moreover mrgin styles will make it more responsive for mobile devices


On Dec 7, 2018 17:18, "Sahil Saha" <notifications@github.com> wrote:

@abishekvashok <https://github.com/abishekvashok> I did the same first but
was requested by @pranav1698 <https://github.com/pranav1698> to do it in
some other way

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/fossasia/meilix-generator/pull/360#issuecomment-445209232>,
or mute the thread

**Comment by sarnava1** [comment]
@pranav1698 I second you.

**Comment by sahilsaha7773** [comment]
Please review @abishekvashok

**Comment by rpotter12** [comment]
@sahilsaha7773 you have to rebase you PR

**Comment by sahilsaha7773** [comment]
Oops, sorry I have messed up all the commits. I am creating another PR.

**Pull request closed without merging**